# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../minised/compile.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

The usual stuff tracked by gcc >= 2.4.x ...

  - Rene Rebe <rene@exactcode.de>

diff -ur sed-1.4/sedcomp.c sed-1.4-fixed/sedcomp.c
--- sed-1.4/sedcomp.c	2003-12-29 22:16:25.000000000 +0100
+++ sed-1.4-fixed/sedcomp.c	2005-02-02 16:04:29.114905024 +0100
@@ -184,11 +184,14 @@
 	XCMD,	H+YCMD,	0,	H+BCMD,	0,	H,	0,	0,
 };
 
+static char *address();
+static int cmdcomp(char);
+static int cmdline(char *);
+
 static void compile(void)
 /* precompile sed commands out of a file */
 {
     char	ccode;
-    static char	*address(), *cmdcomp(char), *cmdline(char *);
 
 	for(;;)					/* main compilation loop */
 	{
--- sed-1.4/Makefile	2003-12-29 23:00:31.000000000 +0100
+++ sed-1.4-fixed/Makefile	2005-02-02 16:13:03.024778848 +0100
@@ -3,10 +3,10 @@
 VERS=$(shell sed <sed.spec -n -e '/Version: \(.*\)/s//\1/p')
 
 sed: sedcomp.o sedexec.o
-	cc $(LFLAGS) sedcomp.o sedexec.o -o sed
+	$(CC) $(LFLAGS) sedcomp.o sedexec.o -o sed
 
 mnsed: mnsed.c
-	cc $(LFLAGS) mnsed.c -o mnsed
+	$(CC) $(LFLAGS) mnsed.c -o mnsed
 
 sedcomp.o: sedcomp.c sed.h
 sedexec.o: sedexec.c sed.h
